.github/workflows/ci.yml

```diff
@@ -0,0 +1,47 @@
+name: CI
+on: [push, pull_request]
+env:
+  CI: true
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        python: [3.5, 3.6, 3.7]
+        os: [ubuntu-latest, macos-latest]
+
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python }}
+          architecture: 'x64'
+
+      - run: python --version
+
+      - name: Cache dependencies
```

Comment:
This is actually optional for me. But we can keep it.

Reply:
I don't mind removing it and in fact it's something that has confused me in the past because of there lines:

https://github.com/StevenBlack/hosts/blob/master/ci/install_conda.sh#L5-L10